lib/broccoli/ember-app.js

```diff
@@ -1341,15 +1321,17 @@ EmberApp.prototype.styles = function() {
     }.bind(this));
   }
 
-  this.import('vendor/addons.css');
-
   var vendorStyles = [];
   for (var outputFile in this._styleOutputFiles) {
+    var isMainVendorFile = outputFile === this.options.outputPaths.vendor.css;
     var headerFiles = this._styleOutputFiles[outputFile];
+    var inputFiles = isMainVendorFile ? ['modules/**/*.css'] : [];
```

Comment:
This is specifically for addon/styles which comes from the addon tree (and was previously being concatted into addons.css (near one of your other comments). It definitely is namespaced to modules/.